src/luaotfload-main.lua

```diff
@@ -29,8 +29,8 @@ local ProvidesLuaModule = {
 }
 
 if luatexbase and luatexbase.provides_module then
-  luatexbase.provides_module (ProvidesLuaModule)
-end  
+    luatexbase.provides_module (ProvidesLuaModule)
+end
```

Comment:
Also a consistency thing: `provides_module` is always indented by two spaces and files are generally split between four space indent and 2 space indent. This also needs cleanup, but more generally and we first need to agree how we want to indent in general (we will probably end up with two spaces).